themes/digital.gov/src/scss/new/guides/_guide-menu-bar.scss

```
@@ -54,6 +43,23 @@
     display: none;
   }
 
+  .dg-guide__menu-bar-link {
+    color: color("black");
```

Alternatively, ink is the default text color for USWDS.

```suggestion
    color: color("ink");
```